plugin/src/py/android_screenshot_tests/pull_screenshots.py

```diff
@@ -180,15 +180,24 @@ def _summary(dir):
 
 def pull_screenshots(process,
                      adb_puller,
+                     perform_pull=True,
                      temp_dir=None,
                      filter_name_regex=None,
                      record=None,
                      verify=None,
                      opt_generate_png=None):
+
+    # Must give a local directory when not pulling
```

Comment:
(we tend to avoid comments inside methods, often you can extract a method and add javadoc for the method when it's really required)